apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -209,18 +209,31 @@ defmodule Astarte.RealmManagement.Queries do
   end
 
   def check_astarte_health(client, consistency) do
-    realms_count_statement = """
-    SELECT COUNT(*)
+    query_schema = """
```

Comment:
Let's keep the naming convention: this is a statement (so `schema_statement`), the query is the thing built from the statement + consistency + other stuff (so `schema_query`)